openshift/Jenkinsfile

```diff
@@ -0,0 +1,86 @@
+pipeline {
+    agent none
+    environment {
+        // Pipeline-wide vars
+        name =  "wps-web"
+        ver = "0.0.1"
+        projTools = "auzhsi-tools"
+        projDev = "auzhsi-dev"
+        projProd = "auzhsi-prod"
+        suffix = "pr-${env.CHANGE_ID}"
+        rcToken="https://chat.pathfinder.gov.bc.ca/hooks/7A8RwSpZPDAqc6uXS/z8nRXX43oifYWELkwJKeLj4e86rtyfvtJdqdvjSE3XoK2sri"
```

Comment:
This doesn't feel safe, I don't think we should be storing a token in source control.

Reply:
@Sybrand We're also no longer using that nuisance Rocket.Chat notification anyway.  Good catch.